lab-abigail/lib/storage.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const debug = require('debug')('http:storage');
+const storage = {};
+
+module.exports = exports = {};
+
+exports.createItem = function(schema, food) {
+  debug('#createItem');
+
+  if(!schema) return Promise.reject(new Error('schema required'));
+  if(!food) return Promise.reject(new Error('food required'));
+  if(!storage[schema]) storage[schema] = {};
+
+  storage[schema][food.id] = food;
+
+  return Promise.resolve(food);
```

Comment:
this is great. moving forward, we may want to resolve certain properties, as opposed to the entire object itself. depends on the scope of the project, and the efficiency of pulling one/two props off vs the entire resource :)